selftests/functional/test_replay_basic.py

```diff
@@ -197,6 +197,16 @@ def test_run_replay_fallbackdir(self):
         expected_rc = exit_codes.AVOCADO_ALL_OK
         self.run_and_check(cmd_line, expected_rc)
 
+    def test_run_replay_and_mux(self):
```

Comment:
True, it is because I developed it on the original tree and then squashed the changes. The full-stevedore-plugin does not use this API at all so I'll remove the handling completely.

With the re-worked plugin we unpickle the new Varianter object with the original stevedore dispatcher, which is already parsed so it's not possible to utilize the `-m`. The same applies for the default_params (manually checked).